controllers/cluster.weave.works/existinginframachine_controller.go

```diff
@@ -483,6 +486,27 @@ func (a *ExistingInfraMachineReconciler) update(ctx context.Context, c *existing
 	if err = a.setNodeProviderIDIfNecessary(ctx, node); err != nil {
 		return err
 	}
+	isMaster := isMaster(node)
```

Comment:
This has moved up in the code to before the 
```go
contextLog.Infof("........................NEW UPDATE FOR: %s...........................", machine.Name)
```
now, does it have to run before other things?